spec/services/persister/post_persister_spec.rb

```diff
@@ -4,25 +4,40 @@
   let(:organization) { Fabricate(:organization) }
   let(:user) { Fabricate(:user) }
   let(:category) { Fabricate(:category) }
-  let(:post) { Fabricate(:post, organization: organization) }
+  let(:post) do
+    Fabricate.build(
+      :offer,
+      organization: organization,
+      user: user,
+      category: category,
+      title: 'Title'
+    )
+  end
+  let(:persister) { ::Persister::PostPersister.new(post) }
 
   describe '#save' do
-    it 'saves the post' do
-      post = Offer.new(organization: organization, user: user, category: category, title: 'Title')
-      persister = ::Persister::PostPersister.new(post)
-
-      persister.save
+    before { persister.save }
 
+    it 'saves the post' do
       expect(post).to be_persisted
     end
+
+    # TODO: write better expectation
+    it 'creates an event' do
+      expect(Event.where(post_id: post.id).first.action).to eq('created')
+    end
   end
 
   describe '#update_attributes' do
-    it 'updates the attributes' do
-      persister = ::Persister::PostPersister.new(post)
-      persister.update_attributes(title: 'New title')
+    before { persister.update_attributes(title: 'New title') }
 
+    it 'updates the resource attributes' do
       expect(post.title).to eq('New title')
     end
+
+    # TODO: write better expectation
+    it 'creates an event' do
+      expect(Event.where(post_id: post.id).first.action).to eq('updated')
```

Comment:
I left this broken spec to remember to add the "updated" business logic in `PostPersister#update_attributes()`